### ISSUE TITLE
Email Form

### ISSUE THREAD

**Issue by valkyrienyanko** [valkyrienyanko/valky#1]
Can someone create an email form that submits to this address `sebastianbelle074@gmail.com` at the very bottom of the website just above the GitHub link.

I just need something simple where the form asks for their name, email?, and a place to put their comment.

Website is hosted on GitHub pages here https://valkyrienyanko.github.io/valky/

**Comment by dcorriveau** [comment]
There isn't any native support for sending out emails from GitHub Pages. However, there are 3rd party alternatives.

One I've noticed is called [FormSpree](https://formspree.io) where it provides you the HTML, and it handles the emailing out.

I have attempted creating an account with your email and a default password, as well as a PR with the code. Let me know when you are available and I can email you the password (so you can have access to the FormSpree account).

Thanks!

**Comment by tulsi-prasad** [comment]
I think an easier alternative to this would be creating one email form using some mailchimp or sendinblue drag and drop editor, and embedding that into your website. It's easy to use for your campaigns and newsletter etc, also.

**Comment by dcorriveau** [comment]
That is true as well if @valkyrienyanko wanted to use newsletters, but I'm not sure if @valkyrienyanko is looking for more than just a simple form...

**Comment by valkyrienyanko** [comment]
Is it possible to configure `formspree.io` so it does not take you to their site to confirm form submission?

**Comment by dcorriveau** [comment]
That should work. Let me give it a try in my commit

**Comment by valkyrienyanko** [comment]
I'm going to look into formkeep.com in the meantime.

**Comment by dcorriveau** [comment]
okay. In any case, i made a commit within my pull request that should handle form redirecting.

It will now stay on the page, and give a alert thanking you for submitting the form.

Give it a try and let me know if you like it.

**Comment by valkyrienyanko** [comment]
When I clicked the send button it refreshed the page and put me at the top, but no alert popped up on the page.

**Comment by valkyrienyanko** [comment]
Maybe the refreshing of the page has something to do with the alert not showing up?

**Comment by dcorriveau** [comment]
It was because I forgot to wrap the jquery code in script.js in a document.ready method (so it would run before anything was loaded, and not link with the form). Oops!

I did notice something however. In order to use Ajax with Formspree.io, you need to be a gold member, which has a fee attached. I didn't notice this till now, so I'm not sure if that's something you'd want...

I am going to test out formsubmit.io, as it looks to be a free alternative in the comments of where i found parts of the code for formspree.

**Comment by dcorriveau** [comment]
in the meantime, I have committed my code that should work once we find a site that allows for Ajax forms to be submitted...

**Comment by valkyrienyanko** [comment]
Oh why did I close this. I guess the PR auto closed it. oops

**Comment by valkyrienyanko** [comment]
Who knew form submissions could be so complicated for static webpages.

**Comment by valkyrienyanko** [comment]
Maybe I should migrate to heroku to host the webpage so it isn't static anymore? I have little knowledge of setting up website servers though..

**Comment by dcorriveau** [comment]
The main thing is that you need a server to handle all of the email items. The one web page cant do it on it's own, so you need to post it somewhere.

The code should be fine to use, we just need to find a site that will let you post the email details to so it can send it out...

Formspree does for a fee, but other than that, I'd have to look when I get home.



Sent from my Bell Samsung device over Canada's largest network.


-------- Original message --------
